### ISSUE TITLE
Publishing to CRAN

### ISSUE THREAD

**Issue by sebastian-c** [stenevang/sftp#3]
Hi, this package seems like it fills a hole in the existing R ecosystem. Have you considered publishing it to CRAN?

**Comment by stenevang** [comment]
I have considered that - however, I'd like to get the package to the level where it supports authentication with keyfiles before I put it on CRAN. 

The other thing I'm not satisfied with is the current solution for recursive directory listing which has bad performance.

**Comment by pbchase** [comment]
Yes, the key-based authentication would be very nice. Do you accept PRs? We're not in a position to contribute now, but perhaps in a 4 months we might have cycles

**Comment by christophvw** [comment]
Could you please publish it to CRAN anyway? We cannot allow our users to install packages from github (and it won't work anyway as we cannot get kerberos proxy auth to work with devtools).

**Comment by gwselke** [comment]
Yes, publishing to CRAN would be great! Even if the package doesn't implement each and every desirable feature yet, having it widely available would help many R users because of increased visibility.
